src/balatro_gym/cards/interfaces.py

```diff
@@ -363,7 +371,7 @@ def deal(self, num: int) -> Sequence[PlayingCard]:
     def destroy(self, cards: Sequence[PlayingCard]) -> None:
         """Destroyed cards are removed permanently. Though, this can only happen to cards in-play."""
         for card in cards:
-            self._cards_remaining.remove(card)
+            self._cards_played.remove(card)
```

Comment:
You also want to have a check otherwise we might hit a value error
```suggestion
            if card in self._cards_played:
                self._cards_played.remove(card)
```